marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-operator.json

```diff
@@ -10,7 +10,8 @@
     "hub-central-custom-reader",
     "hub-central-step-runner",
     "hub-central-saved-query-user",
-    "hub-central-entity-exporter"
+    "hub-central-entity-exporter",
+    "redaction-user"
```

Comment:
Per DHFPROD-6297, "redaction-user" should be assigned to “hub-central-entity-exporter” which is inherited by hub-central-developer,  hub-central-operator & hub-central-explorer)

Reply:
It was not assigned as per Shannon's comment https://project.marklogic.com/jira/browse/DHFPROD-6297?focusedCommentId=188979&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-188979

I'll update the AC.

Reply:
So should it be added to "hub-central-operator" as I don't see that in the story ?

Reply:
It should be since we can run custom steps via HC.
It's mentioned in the AC part of the story.